### DISCUSSION TITLE
[Feature] Manual zoom function

### DISCUSSION THREAD

**Discussion by bassiecom** [johanohly/AirTrail#99] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When I zoom in the homepage is automatically reset in a second to the view with all flights. When you wish to see more details of a region than that is not possible now.

Little addition, the + and - button on the upper left don't work (on different browsers) so maybe this is a issue instead of a feature request

**Comment by johanohly** [comment]
Thank you for this feature request!
I'm not entirely sure why the homepage resets when you zoom in, as the automatic reset should only be triggered when the browser window is resized or when flights are added or removed. The functionality you are describing is not intended behaviour. 
Are you able to provide a video of what is happening on your screen?

**Reply by bassiecom** [reply]
Here it is ...

https://github.com/user-attachments/assets/

**Reply by johanohly** [reply]
Thank you. I can't really see what could cause that, and you're the first one to report this behavior. What browser are you using?

**Reply by bassiecom** [reply]
I have it both on FF and Chrome on my Win11 machine. Now on FF on my Android works good. Very strange. Will look further on my machine for the problem.
